app/controllers/maintenance_tasks/tasks_controller.rb

```diff
@@ -12,7 +12,9 @@ class TasksController < ApplicationController
     # Renders the maintenance_tasks/tasks page, displaying
     # available tasks to users, grouped by category.
     def index
-      @available_tasks = TaskDataIndex.available_tasks.group_by(&:category)
+      tag = params[:tag] || nil
```

Comment:
```suggestion
      tag = params[:tag]
```

params[:tag] will be nil if nothing is passed in